File: CompSci 1/Unit 14, Iteration - while/RepeatWords/RepeatWords.java
import java.util.Scanner;

public class RepeatWords {
    public static void main(String[] args)
    {
        Scanner sc = new Scanner(System.in);

        String word = "";
        int count = 0;

        System.out.println("Enter words, type 'stop' or 'exit' to stop: \n");
        while(!word.equalsIgnoreCase("stop") && (!word.equalsIgnoreCase("exit"))) {
            word = sc.next();
            count++;
        }

        System.out.println("You entered " + (count - 1) + " words.");
    }
}

File: CompSci 1/Unit 14, Iteration - while/RepeatWords/RepeatWords.kt
import java.util.Scanner

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`)

    var word = ""
    var count = 0

    println("Enter words, type 'stop' or 'exit' to stop: \n")
    while (!word.equals("stop", ignoreCase = true) && !word.equals("exit", ignoreCase = true)) {
        word = sc.next()
        count++
    }

    println("You entered " + (count - 1) + " words.")
}